app.js

```diff
@@ -1 +1,323 @@
 // Cookies needed per location projections
+
+var hours = ['6am', '7am', '8am', '9am', '10am', '11am', '12pm', '1pm', '2pm', '3pm', '4pm', '5pm','6pm', '7pm', '8pm'];
```

Comment:
You want the sales during the open hours. Note that the store isn't open from 8-9pm, so no need to include the 8pm hour.